packages/mobile/src/account/__snapshots__/Account.test.tsx.snap

```diff
@@ -598,6 +598,83 @@ exports[`Account renders correctly 1`] = `
             </View>
           </View>
         </View>
+        <View
```

Comment:
Why did this file change?

Reply:
DataSaver was behind a feature flag so didn't show up in Account before